Building LwIP web server with CDC-NCM throws an error

### Operating System

MacOS

### Board

STM32F103C8T6 Blue Pill

### Firmware

examples/device/net_lwip_webserver

### What happened ?

Building LwIP web server example for STM32F103C8T6 Blue Pill with CDC-NCM stack throws an error

### How to reproduce ?

Steps to reproduce it:
1. Go to examples/device/net_lwip_webserver/src/tusb_config.h, and modify `CFG_TUD_ECM_RNDIS` to `1`
2. Go to examples/device/net_lwip_webserver and build: `make BOARD=stm32f103_bluepill all`

### Debug Log as txt file

```
(base) macos@Eka-MacBook-Pro net_lwip_webserver % make BOARD=stm32f103_bluepill all  
AS startup_stm32f103xb_asm.o
CC dcd_stm32_fsdev.o
CC system_stm32f1xx.o
.
.
.
CC ecm_rndis_device.o
CC ncm_device.o
/Users/macos/Developer/tinyusb/src/class/net/ncm_device.c: In function 'netd_control_xfer_cb':
/Users/macos/Developer/tinyusb/src/class/net/ncm_device.c:395:43: error: cast discards 'const' qualifier from pointer target type [-Werror=cast-qual]
  395 |         tud_control_xfer(rhport, request, (void*)&ntb_parameters, sizeof(ntb_parameters));
      |                                           ^
compilation terminated due to -Wfatal-errors.
cc1: all warnings being treated as errors
make: *** [../../rules.mk:131: _build/stm32f103_bluepill/obj/src/class/net/ncm_device.o] Error 1
```

### Screenshots

_No response_

### I have checked existing issues, dicussion and documentation

- [X] I confirm I have checked existing issues, dicussion and documentation.

--- a/src/class/net/ncm_device.c
+++ b/src/class/net/ncm_device.c
@@ -392,7 +392,7 @@ bool netd_control_xfer_cb(uint8_t rhport, uint8_t stage, tusb_control_request_t 
 
       if (NCM_GET_NTB_PARAMETERS == request->bRequest)
       {
-        tud_control_xfer(rhport, request, (void*)&ntb_parameters, sizeof(ntb_parameters));
+        tud_control_xfer(rhport, request, (void*)(uintptr_t) &ntb_parameters, sizeof(ntb_parameters));
       }
 
       break;

